mne/preprocessing/stim.py

```diff
@@ -67,3 +79,108 @@ def eliminate_stim_artifact(raw, events, event_id, tmin=-0.005,
         elif mode == 'window':
             raw[picks, first_samp:last_samp] = data * window[np.newaxis, :]
     return raw
+
+
+def fix_stim_artifact(inst, events=None, event_id=None, tmin=-0.2,
+                      tmax=-0.1, mode='linear'):
```

Comment:
the default tmin and tmax do not make sense. The stim artifact is after stim to tmin=0. and tmax=0.01 makes way more sense.